Sources/FaunaDB/QueryResult.swift

```diff
@@ -1,5 +1,12 @@
 import Foundation
 
+/**
+    Represent the result of an asynchronous query executed in a FaunaDB server.
+
+    - Note: All methods available to handle `QueryResult` success or failure.
+    will optionally receive a `DispatchQueue`. The only `DispatchQueue` allowed
+    to update the UI is `DispatchQueue.main`.
```

Comment:
This is just general iOS / UI programming right that the reader should know right? In that case, maybe say: "Remember, the only `DispatchQueue`" or, on a new line: "Reminder: The only `DispatchQueue`..."